src/Microsoft.AspNetCore.Mvc.WebApiCompatShim/Conventions/IUseWebApiActionConventions.cs

```diff
@@ -3,6 +3,9 @@
 
 namespace Microsoft.AspNetCore.Mvc.WebApiCompatShim
 {
+    /// <summary>
+    /// Indicates actions without attribute routes in a controller use WebAPI routing conventions.
```

Comment:
Talked offline w/ @rynowak and he corrected my understanding. Will go with "ASP.NET Web API" since this shim is _only_ about compat w/ that product.